src/libs/runtime-utils.js

```diff
@@ -209,7 +213,7 @@ const ephemeralExternalIpAddressCost = ({ numStandardVms, numPreemptibleVms }) =
   return numStandardVms * ephemeralExternalIpAddressPrice.standard + numPreemptibleVms * ephemeralExternalIpAddressPrice.preemptible
 }
 
-export const runtimeCost = ({ runtimeConfig, status }) => {
+export const getRuntimeCost = ({ runtimeConfig, status }) => {
   switch (status) {
```

Comment:
I like the name change here. If you're feeling ambitious you could also update this code to use `Utils.switchcase` 🙏